src/api/config.py

```diff
@@ -23,7 +23,7 @@ def POSTGRES_USER():
 
 
 def POSTGRES_PASSWORD():
-    POSTGRES_PASSWORD = "1250"
+    POSTGRES_PASSWORD = "postgres"
```

Comment:
Credentials should be passed either by command line or a config file. Never hardcoded.